exercises/practice/bank-account/.meta/config.json

```diff
@@ -1,6 +1,65 @@
 {
   "blurb": "Simulate a bank account supporting opening/closing, withdraws, and deposits of money. Watch out for concurrent transactions!",
   "authors": [],
```

Comment:
@ErikSchierboom Empty? Is that allowed?

Here's the commit that adds the exercise: https://github.com/exercism/elixir/commit/0afa70cc5e6affc4f24bf2eb3158a4ca30942fbe#diff-8e71c6ee772e27e4a11c87370b8d2689457a4287af7fb2947fd875093a657d93

Looks like the author doesn't have a github account? I guess that's why it's empty?